header-check/header-check

```diff
@@ -106,7 +124,7 @@ test_header() {
 	esac
 	cat configure.ac
 	autoconf
-	./configure && rc=0 || rc=1
+	./configure "$configure_opts" && rc=0 || rc=1
```

Comment:
i think you probably _don't_ want quoting here.  its tricky. you want to support both:

```"-DMENU=taco tuesday"```

and:
```"-DFOOD=apple -Dfruit=banana"```

and then the combination of both of those together.